alvr/server/cpp/alvr_server/HMD.h

```diff
@@ -6,6 +6,8 @@
 #include <memory>
 #ifdef _WIN32
 #include "platform/win32/OvrDirectModeComponent.h"
+#else
```

Comment:
This should be `#elif __linux__` just in case SteamVR we want to support other targets

Reply:
It also matches the style elsewhere